controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -448,6 +458,46 @@ bool master_thread::autoconfig_wsc_calculate_keys(std::shared_ptr<ieee1905_1::tl
     return true;
 }
 
+/**
+ * @brief autoconfig global authenticator attribute calculation
+ * 
+ * Calculate authentication on the Full M1 || M2* whereas M2* = M2 without the authenticator
+ * attribute.
+ * 
+ * @param m1 WSC M1 TLV
+ * @param m2 WSC M2 TLV
+ * @param authkey authentication key
+ * @return true on success
+ * @return false on failure
+ */
```

Comment:
Do we put doxygen comments in the header or in the source? Or maybe in the header for external (i.e.public) function and in the source for private functions?

Reply:
I tend to follow the file I'm editing so in this case, the cpp file. I prefer them in the header so what we did in the encryption impl shouldn't be changed, but we can open a refractor task for moving all doxygen comments to the headers.